src/components/seeds/database/index.tsx

```diff
@@ -201,8 +201,9 @@ export default function Database(props: DatabaseProps): JSX.Element {
     const stage = query.get('stage');
     const facilityId = query.get('facilityId');
     let newSearchCriteria = searchCriteria || {};
-    if (stage) {
-      if (ACCESSION_STATES.indexOf(stage) !== -1) {
+    if (stage || query.has('stage')) {
```

Comment:
if stage is equal to query.get('stage'); why is this or necessary?

Reply:
query.get('stage') will capture `stage=<something>` query.has('stage') will capture if the key stage exists - even if there is no value. query.get('stage') will only capture value.

Reply:
Oh, ok, thanks for explaining that